.github/workflows/budibase_ci.yml

```diff
@@ -22,44 +22,51 @@ jobs:
   lint:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
       - name: Use Node.js 14.x
-        uses: actions/setup-node@v1
+        uses: actions/setup-node@v3
         with:
           node-version: 14.x
+          cache: "yarn"
       - run: yarn
       - run: yarn lint
 
   build:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
         with:
           submodules: true
           token: ${{ secrets.PERSONAL_ACCESS_TOKEN }}
       - name: Use Node.js 14.x
-        uses: actions/setup-node@v1
+        uses: actions/setup-node@v3
         with:
           node-version: 14.x
+          cache: "yarn"
       - run: yarn
-      - run: yarn bootstrap
       - run: yarn build
 
   test:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
         with:
           submodules: true
           token: ${{ secrets.PERSONAL_ACCESS_TOKEN }}
       - name: Use Node.js 14.x
-        uses: actions/setup-node@v1
+        uses: actions/setup-node@v3
         with:
           node-version: 14.x
+          cache: "yarn"
       - run: yarn
-      - run: yarn bootstrap
-      - run: yarn build
+      - run: yarn build --scope=@budibase/types --scope=@budibase/shared-core --scope=@budibase/string-templates
```

Comment:
We didn't used to have to build to run the tests, where we should be able to use source mappings instead. Is there a way we can revert to this behaviour? looks like it was added here

https://github.com/Budibase/budibase/commit/80002873aaed2f4ea7933efe2487bf4f9196b96d
https://github.com/Budibase/budibase/commit/96f2e830728c12ee7ad5ac2ff17c96fe69d83e3c

Reply:
This is required to build the shared-core for the js projects (builder and others) to run. I don't think we can use jest paths for this with js to ts

Reply:
I see! One option could be to run the ui tests with ts-jest using the swc or esbuild plugins, would be very fast then. 
Maybe when we move to esbuild everywhere it's not much of a problem though